html/includes/graphs/application/ups-nut_voltage_battery.inc.php

```diff
@@ -48,7 +51,7 @@
         $i++;
     }
 } else {
-    echo "file missing: $rrd_filename";
+    echo "file missing: ".$rrd_filename;
```

Comment:
I don't mind the echo, but this is an odd change.  I think it is easier to read in the previous form.

FYI, variables inside double quotes are expanded, variables inside single quotes are not.